src/platforms/windows/daemon/windowstunnellogger.cpp

```diff
@@ -107,6 +108,10 @@ void WindowsTunnelLogger::process(int index) {
 
   QByteArray message((const char*)data + RINGLOG_TIMESTAMP_SIZE,
                      RINGLOG_MESSAGE_SIZE);
+  int nullIndex = message.indexOf(0x0);
```

Comment:
Oooh! Good catch! We could also have used `QString::fromUtf(const char *str, int size = -1)` which would simplify this even further.

Reply:
Correct.  I'll post an update doing the proper conversion.

Reply:
Turns out it isn't.  Left in the current truncation.